packages/amplify-cli/tslint.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This doesn't seem to be executed by the top-level linter. Please make sure to make it a part of the top-level lint command.
There are a bunch of places where the indentation/style is not the same as the rest of the CLI.Make sure to follow the AirBnb linting rules - as we do for the other parts of the CLI.

Reply:
will get on the lint part today, we have a ts and js mixed project.